docs/faq/metadata.md

```diff
@@ -37,7 +37,7 @@ Geographic locations can be broken down, for example, into `region`, `country`,
 
 It is important that these are spelled consistently.
 
-If you want to include locations where augur doesn't know the lat-long values, you can include them - see how [here](./lat_longs.html).
+If you want to include locations where augur doesn't know the lat-long values, you can include them - see how [here](https://docs.nextstrain.org/en/latest/guides/bioinformatics/lat_longs.html).
```

Comment:
I wish we didn't have to include absolute URLs here and could use Intersphinx references so they stay valid.

But since we can't (right?), maybe let's at least `s{en/latest}{page}` this URL to more future proof it against changes to our RTD config for the main doc project.

Reply:
I don't think we can use Intersphinx references on the Markdown, but I might as well convert to reStructuredText in this PR to make use of it.

Reply:
Done: 8c335980f045056e8eaa7c484dfff95029b48368